app/model/toolbar/CircleSelectionTool.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * This class is the view model for the CircleSelectionToolbar.
+ */
+Ext.define('CpsiMapview.model.toolbar.CircleSelection', {
```

Comment:
The class name should reflect the the JS file incl. its path:

Due to the path of the source file the class should be named the following: `CpsiMapview.model.toolbar.CircleSelectionTool`

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/177/commits/b79cde00e7b4d84cc737d59866c7570cad768ee5